### PR TITLE
[onecc-docker] Add argparse to onecc-docker

### PR DESCRIPTION
**Pull request by morumoru**
This commit add argparse to onecc-docker to handle with GitHub token

ONE-DCO-1.0.-Signed-off-by: Junsu Kim <xofkdqkqh@naver.com>
Co-authored-by: Dongwan Kim <asdf134652@gmail.com>
Co-authored-by: seunghui-lee <hithere1012@naver.com>

/cc @Samsung/ootpg_docker

### CODE REVIEW

File: compiler/onecc-docker/onecc-docker [hunk 1]
```diff
@@ -46,9 +47,26 @@ def main():
...
+
+    args, onecc_arguments = parser.parse_known_args()
+    authorization_token = args.token
+
     LATEST_URL = "https://api.github.com/repos/Samsung/ONE/releases/latest"
     try:
-        response = requests.get(LATEST_URL)
+        headers = {}
+        if authorization_token:
+            headers = {"Authorization": "Bearer {}".format(authorization_token)}
```
Comment (jyoungyun) [inline compiler/onecc-docker/onecc-docker:68]: It would be better to take it out of the `try` statement except for the `requests` function.

Comment (morumoru) [reply compiler/onecc-docker/onecc-docker]: Thank you for advice! I took out `header` related part from `try` statement.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**